zingolib/src/lightclient/test_features.rs

```diff
@@ -0,0 +1,18 @@
+use super::*;
+
+impl LightClient {
+    pub fn is_mobile_target() -> bool {
+        #[cfg(not(any(target_os = "ios", target_os = "android")))]
+        {
+            true
+        }
+        #[cfg(any(target_os = "ios", target_os = "android"))]
+        {
+            false
+        }
+    }
+}
+#[tokio::test]
+async fn os_target() {
+    assert!(LightClient::is_mobile_target());
```

Comment:
Why does this test not fail? This should evaluate as false and fail the assert?

Reply:
just like you told me, we are building for a mobile target when we run the tests. so this passes

Reply:
CI builds on a non-mobile target? i dont understand...